src/Params/WaveTableFwd.h

```diff
@@ -0,0 +1,63 @@
+/*
+  ZynAddSubFX - a software synthesizer
+
+  WaveTable.h - WaveTable forward declarations
+  Copyright (C) 2020-2022 Johannes Lorenz
+  Author: Johannes Lorenz
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+*/
+
+#ifndef WAVETABLEFWD_H
+#define WAVETABLEFWD_H
+
+#include <cassert>
```

Comment:
No change required either way, but purely out of curiosity is cassert needed for static_assert? I figured that was a builtin.

Reply:
I never needed to include `cassert` when using `static_assert`, and the `assert` header does not even offer `static_assert`.

The include is here just for the use of the "normal" `assert`s.